### PR TITLE
Don't ovewrite "fulfilled" properties from app in BETS API

### PR DESCRIPTION
**Pull request by proteusvacuum**
@nickpell 

We were overwriting properties set by the app with other data from BETS.

### CODE REVIEW

**Comment by proteusvacuum** [comment]
FYI @esoergel

**Review by nickpell — APPROVED** [review]

File: custom/enikshay/integrations/bets/tests/test_bets_updates.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def test_update_voucher_success(self):
         }]})
```
Comment (esoergel) [inline custom/enikshay/integrations/bets/tests/test_bets_updates.py]: :+1: on the rename

File: custom/enikshay/integrations/bets/views.py [hunk 2]
```diff
@@ -69,8 +69,9 @@ def properties(self):
         if self.status == SUCCESS:
             return {
                 'state': 'paid',
-                'amount_fulfilled': self.amount,
-                'date_fulfilled': self.paymentDate.isoformat(),
+                'amount_paid': self.amount,
+                'date_paid': self.paymentDate.date().isoformat(),
+                'time_paid': self.paymentDate.time().isoformat(),
```
Comment (esoergel) [inline custom/enikshay/integrations/bets/views.py]: This seems odd - what's the deal with splitting this up?  I'm sure you've got your reasons, just curious.

**Review by esoergel — APPROVED** [review]

File: custom/enikshay/integrations/bets/views.py (hunk 2 above)
Comment (proteusvacuum) [reply custom/enikshay/integrations/bets/views.py]: we don't deal with date-times well (or at all) on formplayer, so if this data needs to be shown anywhere / filtered in the app, it's better to just do it on "date" where we can actually cast to a date.

**Pull request merged**